tests/mail/test_sendtestemail.py

```diff
@@ -2,32 +2,78 @@
 
 from django.core import mail
 from django.core.management import call_command
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, override_settings
 
 
+@override_settings(
+    ADMINS=(('Admin A', 'admin_a@example.com'), ('Admin B', 'admin_b@example.com')),
+    MANAGERS=(('Manager A', 'manager_a@example.com'), ('Manager B', 'manager_b@example.com')),
+)
 class SendTestEmailManagementCommand(SimpleTestCase):
+
     """
     Test the sending of a test email using the `sendtestemail` command.
     """
 
-    def test_send_test_email(self):
+    def test_single_receiver(self):
         """
         The mail is sent with the correct subject and recipient.
         """
-        recipient = "joe@example.com"
-        call_command("sendtestemail", recipient)
+        recipient = 'joe@example.com'
+        call_command('sendtestemail', recipient)
         self.assertEqual(len(mail.outbox), 1)
         mail_message = mail.outbox[0]
         self.assertEqual(mail_message.subject[0:15], 'Test email from')
         self.assertEqual(mail_message.recipients(), [recipient])
 
-    def test_send_test_email_with_multiple_addresses(self):
+    def test_multiple_receivers(self):
         """
         The mail may be sent with multiple recipients.
         """
-        recipients = ["joe@example.com", "jane@example.com"]
-        call_command("sendtestemail", recipients[0], recipients[1])
+        recipients = ['joe@example.com', 'jane@example.com']
+        call_command('sendtestemail', recipients[0], recipients[1])
         self.assertEqual(len(mail.outbox), 1)
         mail_message = mail.outbox[0]
         self.assertEqual(mail_message.subject[0:15], 'Test email from')
         self.assertEqual(mail_message.recipients(), recipients)
+
+    def test_manager_receivers(self):
+        """
+        The mail should be sent to the email addresses specified in
+        settings.MANAGERS.
+        """
+        call_command('sendtestemail', '--managers')
+        self.assertEqual(len(mail.outbox), 1)
+        mail_message = mail.outbox[0]
+        self.assertEqual(mail_message.recipients(), [
+            'Manager A <manager_a@example.com>',
+            'Manager B <manager_b@example.com>'
```

Comment:
please include trailing comma in these lists (if more items are added, then we don't have to modify the last line again)